### DISCUSSION TITLE
Can we use multiple data-website-id's?

### DISCUSSION THREAD

**Discussion by dommholland** [umami-software/umami#2045] [Q&A]
Can we use multiple data-website-id's like this:

```
<script async 
src="https://analytics.trady.com/script.js" 
data-website-id="256c904e-7d34-4fd6-9777-111734ba45aa,72833686-1592-4113-b0a0-f5a27b6ae195"
></script>
```

We use this with our current plausible setup so we can record the traffic against two website records, like this: 
https://plausible.io/docs/plausible-script#can-i-send-stats-to-multiple-dashboards-at-the-same-time

Thanks!

**Comment by FHU-yezi** [comment]
Maybe just copy and paste the same script tag, and replace one of them with a different website id?

**Reply by dommholland** [reply]
considered this. although, if i then used umami.track() which website would it be posting the event too, need it posting to both.
